core/amber/src/main/resources/application.conf

```diff
@@ -3,6 +3,9 @@ constants {
     logging-queue-size-interval = 30000
     num-worker-per-node = 2
     data-volume-per-node = 10
+    reshape-skew-handling-enabled = false
+    reshape-eta-threshold = 100
+    reshape-tau-threshold = 100
```

Comment:
please use a separate config object. I suggest the following:

```
optimization {
    workload-reshape {
        skew-handling-enabled = false
        skew-detection-interval = 500
        eta-threshold = 100
        tau-threshold = 100
   }
}
```

Reply:
also do we want to add `monitoring` to config?

Reply:
Yes. I did something similar in PR #1414 . So, I will resolve it here because it is taken care of there. For now, I don't see any need to add monitoring to config.